lambdas/build-user-identity/src/main/java/uk/gov/di/ipv/core/builduseridentity/BuildUserIdentityHandler.java

```diff
@@ -195,12 +199,22 @@ private void sendIdentityIssuedAuditEvent(
             ContraIndicators contraIndicators,
             UserIdentity userIdentity)
             throws SqsException {
+
+        Map<String, ContraIndicatorConfig> configMap = configService.getContraIndicatorConfigMap();
+        var auditEventReturnCodes =
+                userIdentity.getReturnCode().stream()
```

Comment:
Is the property called returnCode on UserIdentity, instead of returnCodes, because it'd break some agreed schema?

Reply:
In the user identity bundle that's returned to core, it looks (something, working from memory) like this:
```
{
...
"https://vocab.account.gov.uk/v1/returnCode": [{"code": "X"}, {"code": "Y"}]
...
}
```

So the return code, can contain mutiple, a single, or no actual codes.

So to answer your question, sort of yes. The return code is all the codes, but we override the property name to something more complicated anyway.

Reply:
Thank you for the run-through!